_config.stage.yml

```diff
@@ -0,0 +1 @@
+url: "https://devdocs.magedevteam.com"
```

Comment:
Does this file fix link checking on the Jenkins staging build? Seems a bit out of scope for this PR, but if it works, great.

Reply:
For the fix, we need a `baseurl` without `url`.  That's why I moved the URL in the separate file. We cannot do it in the CLI like with `--baseurl`.
But seems that staging works ok with `baseurl` only, so we don't need this anymore.

Reply:
With no URL, redirects use URL from the `_config.yml`.
So, we will still need `_config.stage.yml` to override the main config.